tests/functional/dynamodb/test_table.py

```diff
@@ -22,20 +24,93 @@ class TestTableResourceCustomizations(unittest.TestCase):
 
     def setUp(self):
         self.resource = boto3.resource('dynamodb', 'us-east-1')
+        self.table = self.resource.Table('mytable')
```

Comment:
For context, was this a general optimization for the tests or was it needed for something specific with this change?

Reply:
Not required, just code style. I would have added the third duplication (triplication?) of this line, seemed like a good time to move it to setup.